distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso

```diff
@@ -260,4 +260,4 @@ java_aggregator name column =
         Shortest c _ -> ShortestOrLongestAggregator.new name c.java_column ShortestOrLongestAggregator.SHORTEST
         Longest c _ -> ShortestOrLongestAggregator.new name c.java_column ShortestOrLongestAggregator.LONGEST
         Concatenate c _ join prefix suffix quote -> ConcatenateAggregator.new name c.java_column join prefix suffix quote
-        _ -> Error.throw (Invalid_Aggregation.Error name -1 "Unsupported aggregation")
+        unknown -> Error.throw (Invalid_Aggregation.Error name [] "Unsupported aggregation: "+unknown.to_display_text)
```

Comment:
The check has discovered a type error here :)

![image](https://github.com/user-attachments/assets/07e7d38e-18a0-44d8-ba9f-e523a0ba6237)